oneclick.py

```diff
@@ -319,17 +327,13 @@ def get_ebook_availability_info(self, media_type='ebook'):
             resplist = response.json()
         except Exception, e:
             raise BadResponseException(url, "OneClick availability response not parseable.")
-
-        if not resplist:
-            raise IOError("OneClick availability response not parseable - has no resplist.")
-
```

Comment:
It's legitimate for this API call to return an empty list, if a collection is actually empty.